### DISCUSSION TITLE
Question about font files

### DISCUSSION THREAD

**Discussion by endervad** [GpointChen/FFXIVChnTextPatch-GP#144] [Q&A] [answered]
Hi! I'm the head of FFXIV Russian fan-localization team. Our project is currently about 6 months old now and we're having an on-going issue with the game not having Russian letters in some fonts, so we'd like to add them by ourselves.
We currently don't have people who can reverse-engineer files. I've tried to do that by myself with FDT files and I did roughly figure some stuff out, e.g. part of FTHD section. However it's far from enough, and I got stuck pretty early on. That's where I stumbled upon your project.
I did try to compare your modded FDT files with the original (incl. textures) but there are so many changes that I'm afraid I'm still lost. So, would you mind to share your knowledge of these files and how to properly insert desired characters? Maybe you have some research files left over?

**Comment by GpointChen** [comment, marked as answer]
Here is a [github project that makes font files for FFXIV](https://github.com/Soreepeong/XivAlexander).

I think it may help!

**Reply by endervad** [reply]
Thanks a huge lot, I'll definitely check it out :)
